psamm/commands/table.py

```diff
@@ -0,0 +1,179 @@
+
+
+
+from ..command import Command
+import logging
+import xlsxwriter
+
+logger = logging.getLogger(__name__)
+class ExportTableCommand(Command):
+    """Export parts of the model as tab separated tables"""
+
+
+    @classmethod
+    def init_parser(cls, parser):
+        parser.add_argument(
+            'export', metavar='export_type', help='reactions, compounds, medium, limits')
+
+    def run(self):
+        model = self._model
+        export_type = self._args.export
+        if export_type == 'reactions':
+            reaction_export(model)
+        if export_type == 'compounds':
+            compound_parser(model)
+        if export_type == 'medium':
+            media_parser(model)
+        if export_type == 'limits':
+            limits_parser(model)
+        if export_type == 'metadata':
+            metadata_parser(model)
+        if export_type == 'xlsx':
+            xlsx_writer(model)
+
+
+def reaction_export(model):
+    property_set = set()
+    for j in model.parse_reactions():
+        for i, k in j._properties.iteritems():
+            property_set.add(i)
+    property_list = []
+    for i in property_set:
+        property_list.append(i)
```

Comment:
Here you can simply do `property_list = list(property_set)` instead of these three lines.